pages/employeeProfile.js

```diff
@@ -117,19 +118,28 @@ const employeeProfile = (props) => {
     setListOpen(false);
   };
 
+  const formatDate = (d) => {
+    return `${d.getFullYear()}-${d.getMonth() + 1}-${d.getDate()}`;
+  };
+
+
   return (
     <div>
       <Head>
         <title>Employee Profile</title>
         <link rel="icon" href="/atc-logo.png" />
       </Head>
+      
 
       <Navbar pageTitle="Employee Profile">
+        <Link href="/employeeSearch">
+          <Button>Go Back</Button>

```

Comment:
Are we using the button just for styles? if not we don't need it, can just have <Link href="/employeeSearch">Go Back</Link>